src/sentry/integrations/msteams/webhook.py

```diff
@@ -194,6 +194,7 @@ def handle_team_member_added(self, request: Request):
             "team_id": team["id"],
             "team_name": team["name"],
             "service_url": data["serviceUrl"],
+            "tenant_id": channel_data["tenant"]["id"],
```

Comment:
Is `channel_data` guaranteed to have `tenant` and then `id`? You might do `.get` to be safe, but I'm not sure what's normal.

Reply:
Yes, we expect the request data from MS Teams to have `channelData.tenant.id` and if it does not we want it to fail. So this would raise a KeyError right? Is that the right way to handle this?

Reply:
And, we already do this kind of unguarded access in other places in this webhook like [this](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/msteams/webhook.py#L316-L317) for example. So if we haven't run into issues, then I assume we can rely on it.